### PR TITLE
chore: Update README file for Mac OS Monterey and Apple Silicon Macs

### PR DESCRIPTION
**Pull request by frankchn**
## Problem
README documentation is not updated for Apple Silicon Macs on Mac OS Monterey for the v2 branch.

## Solution
Update README file in the FormV2 branch for Mac OS Monterey and Apple Silicon. Also updates the `nan` package to 2.15 to resolve a build error on Apple Silicon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +69,7 @@ Run the following shell command to build the Docker image from scratch. This wil
 npm run dev
 ```
 
-After the Docker image has finished building, the application can be accessed at [localhost:5000](localhost:5000).
+After the Docker image has finished building, the React application can be accessed at [localhost:3000](localhost:3000).
```
Comment (karrui) [inline README.md:72]: May want to also add a note on the port for the server:

```
The server can be accessed at `localhost:5000`.
```

**Review by karrui — COMMENTED** [review]
you may have used `npm install` with `npm` version > 6, which means the package-lock is now using version 2. May want to use npm v6 in this repo (until we get to upgrading the lockfile version in the repo)

**Comment by frankchn** [comment]
@karrui ah you are right, updated to install with node 14 / npm 6. PTAL again :)

File: README.md (hunk 1 above)
Comment (frankchn) [reply README.md]: done

**Review by karrui — APPROVED** [review]
lgtm

**Pull request merged**